data/scripts/iscsid-setup.sh

```diff
@@ -0,0 +1,3 @@
+sed -i 's/node.session.timeo.replacement_timeout/# node.session.timeo.replacement_timeout' /etc/iscsi/iscsid.conf
+sed -i 's/node.conn[0].timeo.noop_out_interval/# node.conn[0].timeo.noop_out_interval' /etc/iscsi/iscsid.conf
+sed -i 's/node.conn[0].timeo.noop_out_timeout/# node.conn[0].timeo.noop_out_timeout' /etc/iscsi/iscsid.conf
```

Comment:
The sed commands are lacking the final `/`. The dots would technically need to be escaped so they don't match any char, but this won't make a difference in practice. Bonus nit-pick: Those three sed executions could be compressed into one with three `-e` parameters.  ;-)

Reply:
And I can write an entire perl script in one line as well.

Reply:
Nobody said it needs to be one line. ;-) It's not important, of course. We still need those slashes though.